core/templates/pages/learner-dashboard-page/goals-tab.component.ts

```diff
@@ -191,6 +192,12 @@ export class GoalsTabComponent implements OnInit {
 
   toggleThreeDotsDropdown(index: number): void {
     this.showThreeDotsDropdown[index] = !this.showThreeDotsDropdown[index];
+    if (this.indexselected !== index) {
+      this.indexselected = index;
+    } else {
+      this.indexselected = -1;
+    }
+    return;
```

Comment:
Change the name and the name case accordingly.

Reply:
Do you want to say that i should change the variable name to topicIndexSelected

Reply:
I have done the required changes. changed "indexselected " to "indexOfSelectedTopic ".